### DISCUSSION TITLE
GHelper and MyAsus not limiting battery charge

### DISCUSSION THREAD

**Discussion by Mitsuoka123** [seerge/g-helper#3474] [Q&A] [answered]
I have got a new G16 G614JIR a week or two ago and I have installed GHelper that worked fabulously on my G15 G513RM.

I want to limit my battery charge to 80%, but it is not working, from resetting to trying to download MyAsus to set the limit through it, but nothing limits my battery charge.

Also, in customization.ini it appears 80%, so it is not the culprit

![ghelper-and-myasus-not-limiting-battery-charge-v0-ziqfcnx4oe6e1](https://github.com/user-attachments/assets/c7ba2ec0-911e-478e-9d3a-83ba20120374)

I have also looked at GHelper FAQ, but I don't understand what is Extra settings and Stop button

![ghelper-and-myasus-not-limiting-battery-charge-v0-8rg5p5edoe6e1](https://github.com/user-attachments/assets/b429c0f1-b1d2-464c-8879-8f29eaebabda)

This is my GHelper. Also, it is already on 0.199, but it says everyday that I need to update it, don't know why..

![ghelper-and-myasus-not-limiting-battery-charge-v0-r4pw7h1moe6e1](https://github.com/user-attachments/assets/3e4ce1e1-d175-47ca-bcd3-45a930702c28)

Anything more I can try?

**Comment by seerge** [comment, marked as answer]
@Mitsuoka123 hello, 

If also MyASUS not able charge limit, most likely it's some kind of firmware hiccup (as this is firmware that actually handles that). You can try doing a [Hard reset](https://github.com/seerge/g-helper/wiki/Troubleshooting#hard-reset--embedded-controller-ec-reset) to see if it helps.

The `Extra` settings is literally button with a gearbox saying Extra on your screenshot. You may want to stop Asus services there if you don't use MyASUS as otherwise they may override charge limit set by G-Helper before.

On your screenshot app is not updated to 0.199 clearly. It shows a new version available. You can download a new release and replace old exe with a new one :)  Make sure that you don't accidentally have like multiple copies of GHelper.exe placed in different folders, and you are replacing the one is set to run on startup.

**Reply by Mitsuoka123** [reply]
I feel so dumb right now... as it was in Laptop Keyboard section, I totally omitted it. Let's see if that changes something and if not, I will try hard reset. 

About 0.199 update, I thought I did it right, but after your answer, I searched for other copy of GHelper.exe, and there was one in temp folder.

Discharging right now, and will get back with update later. Thanks!

**Reply by seerge** [reply]
@Mitsuoka123 i would suggest to place GHelper.exe in some more reasonable folder, and to make sure that copy is scheduled to run on startup - to uncheck / check `Run on Startup` checkbox. It re-creates startup task pointing to the actual app location.

**Reply by Mitsuoka123** [reply]
I have it in Documents folder and was replacing that one all the time, but somehow probably I did something weird on first download a few days ago and it got assigned to a temp folder one.

**Reply by Mitsuoka123** [reply]
After some testing, the problem was solved after I did EC Reset and reinstalled Asus and Armoury Crate Control Interfaces, something went wrong there with the first installation.

Thanks for your help.

**Reply by seerge** [reply]
@Mitsuoka123 glad to hear
